### PR TITLE
Update mobiles.yml

### PR DESCRIPTION
**Pull request by AtomCZ**
updated and added new devices

### CODE REVIEW

**Comment by mimmi20** [comment]
@AtomCZ Could you some test cases for these new devices?

**Comment by Findus23** [comment]
Ideally add an example user agent for every device to `Tests/fixtures` similar to e.g. #5967

**Comment by AtomCZ** [comment]
New or updated devices:

K33a48 | K6
K33a42 | K6 Power
K10a40 | C2
A6020a40 | K5
A7020a40 | K5 Note
A6020a46 | K5 Plus
TB3-730X | Tab3 7
P1a42 | P1
P2a42 | P2
K53a48 | K6 Note
TB-X103F | Tab 10
Z90a40 | Vibe Shot
A2020a40 | Vibe C
PMP7280C3G QUAD | Prestigio Multipad 4 Ultra
SM-J610FN | J6+
SM-T595 | Tab A 10.5
SM-A202F | A20e
LUA-L21 |  
LYO-L21 | Y6 II Compcat
LYO-L01 | Y6 II Compcat
SCL-L21 | Y6

File: regexes/device/mobiles.yml [hunk 1]
```diff
@@ -3986,39 +3986,39 @@ Huawei:
...
+    - regex: '(?:Honor_|HW-)?DUA-L22[);\/ ]'
       model: 'Honor 7S'
-    - regex: '(?:Honor_|HW-)?NEM-(?:AL10|L51|UL10)[);/ ]'
+    - regex: '(?:Honor_|HW-)?NEM-(?:AL10|L51|UL10)[);\/ ]'
       model: 'Honor 5C'
-    - regex: '(?:Honor_|HW-)?NEM-(?:L22|TL00H)[);/ ]'
+    - regex: '(?:Honor_|HW-)?NEM-(?:L22|TL00H)[);\/ ]'
       model: 'Honor 5C Dual SIM'
-    - regex: '(?:Honor_|HW-)?NEM-L21[);/ ]'
+    - regex: '(?:Honor_|HW-)?NEM-L21[);\/ ]'
```
[the remark this answers is not in the document]
Comment (AtomCZ) [reply regexes/device/mobiles.yml]: Ah, okej, will fix it.

**Pull request closed without merging**